Nodejs/Tests/TestAdapter.Tests/TestProjectFactory.cs

```diff
@@ -17,6 +17,8 @@ public enum ProjectName
             NodeAppWithTestsConfiguredOnProject,
             NodeAppWithTestsConfiguredPerFile,
             NodeAppWithAngularTests,
+            reactappwithjesttestsjavascript,
```

Comment:
I would recommend adding a comment for why is this lowercase. Somebody in the future might try to change it.